packages/drupal-vite/README.md

```diff
@@ -0,0 +1,132 @@
+# drupal-vite
+
+A Vite plugin for Drupal integration that makes it easy to connect your Vite-powered frontend with a Drupal backend.
+
+## Installation
+
+```bash
+# Using npm
+npm install drupal-vite
+
+# Using yarn
+yarn add drupal-vite
+
+# Using pnpm
+pnpm add drupal-vite
+
+# Using bun
+bun add drupal-vite
+```
+
+## Usage
+
+In your Vite configuration file (`vite.config.js` or `vite.config.ts`):
+
+```typescript
+import { defineConfig } from 'vite';
+import { drupal } from 'drupal-vite';
+
+export default defineConfig({
+  plugins: [
+    drupal(), // No options needed for basic usage
```

Comment:
Are we sure that no options are required? 

Don't you think we should add the assumption that tokens should be set as an environment variable to use in this way?

Yes is shown below but they can skip that part 😅